setupext.py

```diff
@@ -2165,3 +2165,22 @@ def check(self):
             pass
 
         raise CheckFailed()
+
+
+class OptionalPackageData(OptionalPackage):
+    config_category = "package_data"
+
+
+class Dlls(OptionalPackageData):
+    """
+    On Windows, this packages any DLL files that can be found in the 
+    lib/matplotlib/* directories.
+    """
+    name = "dlls"
+
+    def check_requirements(self):
+        if sys.platform != 'win32':
```

Comment:
I think `sys.platform != 'win32'` is correct here. The `win32` probably refers to the implementation of Python using the Windows API, formerly named Win32. Other Python implementations for Windows, which this patch does not apply to, are using `cygwin` or `cli` (.NET).